recipes/fermikit/test-data/test.fastq

```diff
@@ -0,0 +1,100 @@
+@phiX174_29_494_0_1_0_0_2:0:0_3:0:0_0
```

Comment:
We don't include functional tests in BioConda. Can you come up with a easier example that don't need this external file.